### ISSUE TITLE
[Question] Why relocate logback packages and configuration files

### ISSUE THREAD

**Issue by yiliuTo** [microsoft/ApplicationInsights-Java#3531]
I see the code in [ai.shadow-conventions.gradle.kts](https://github.com/microsoft/ApplicationInsights-Java/blob/main/buildSrc/src/main/kotlin/ai.shadow-conventions.gradle.kts#L38-L47) relocates all loback related packages and configs. So I am confused what is the purpose of relocation and isolation of logback? IMO, since logback is imported in [agent-tooling](https://github.com/microsoft/ApplicationInsights-Java/blob/main/agent/agent-tooling/build.gradle.kts#L37) with all of its dependencies being loaded in the agent classloader, it won't bring conflicts with users' application or other agent dependencies.

https://github.com/microsoft/ApplicationInsights-Java/blob/16a727ac4149eab281fc6c0ab0916116f7320205/buildSrc/src/main/kotlin/ai.shadow-conventions.gradle.kts#L38-L47

**Comment by trask** [comment]
hi @yiliuTo! I believe that you're right and relocation is no longer needed after #3190 (logback was originally in the bootstrap class loader which is why shading was needed before)

**Comment by yiliuTo** [comment]
Thanks @trask  for the explanation.
